app/assets/stylesheets/searchworks4/homepage.css

```
@@ -70,15 +70,49 @@
   }
 }
 
+.what-header {
+  font-weight: 700;
+  color: var(--stanford-cardinal);
+}
+
 .catalog-home-page {
-  .what-header {
-    font-weight: 700;
-    color: var(--stanford-cardinal);
-  }
   @media (min-width: 992px) {
     .tips,
     .features {
       border-inline-start: 1px solid var(--stanford-40-black);
     }
   }
 }
+
+.article-home-page {
+  --tips-col-width: 100%;
+
+  @media (min-width: 1200px) {
+    --tips-col-width: 48%;
+    --tips-height: 28rem;
+
+    .tips ul {
+      height: var(--tips-height);
+    }
+  }
+
+  @media (min-width: 1400px) {
+    --tips-col-width: 48%;
+    --tips-height: 25rem;
+  }
+
+  @media (min-width: 992px) {
+    .tips {
+      border-inline-start: 1px solid var(--stanford-40-black);
+    }
+  }
+
+  .tips {
+    ul {
+      column-gap: 3rem;
+      li {
+        width: var(--tips-col-width);
+      }
+    }
+  }
+}
```

Instead of flex wrapping, can we use multi-column layout (something like:
```suggestion
.tips ul {
  column-count: 2;
}

.tips ul li {
  break-inside: avoid-column;
}
```
)?

---

Ok, cool. I tried columns, but I didn't know about `break-inside` so I saw problems.

---

I only just found out about it and it seems like it might work 🤷‍♂️

---

I've pushed up a new version using columns.